src/sentry/static/sentry/app/views/settings/organizationRelay/modals/add/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t, tct} from 'app/locale';
+import ExternalLink from 'app/components/links/externalLink';
+import {OrderedList} from 'app/components/list';
+import space from 'app/styles/space';
+
+import ModalManager from '../modalManager';
+import Item from './item';
+import Terminal from './terminal';
+
+class Add extends ModalManager {
+  getTitle() {
+    return t('Register Key');
+  }
+
+  getBtnSaveLabel() {
+    return t('Register');
+  }
+
+  getData() {
+    const {savedRelays} = this.props;
+    const trustedRelays = [...savedRelays, this.state.values];
+
+    return {trustedRelays};
+  }
+
+  getContent() {
+    return (
+      <StyledOrderedList>
+        <Item
+          title={tct('Initialize the configuration. [link: Learn how]', {
+            link: (
+              <ExternalLink href="https://docs.sentry.io/product/relay/getting-started/#initializing-configuration" />
+            ),
+          })}
+          subtitle={t('Within your terminal.')}
+        >
+          <Terminal command="./relay config init" />
+        </Item>
+        <Item
+          title={tct(
+            'Go to the file [jsonFile: credentials.json] to find the public key and enter it below.',
```

Comment:
`relay config init` actually outputs the public key it generated:
```shell
[byk@DESKTOP-QII4CIG ~/D/relay]$ relay config init
Initializing relay in C:\Users\byk\Desktop\relay\.relay
Do you want to create a new config?: Yes, create default config
Generated new credentials
  relay id: f99d01b0-2317-4e3a-b6d4-3596ebadbfba
  public key: SdZT8bE4bNJs-e9RiKyupUaWtzR0ToTAYUQuuavWNWc
All done!
```

Reply:
This is true. We chose to mention `credentials.json` since if the config init had already happened it also tells users where to fetch the public key from. 

This would be something to consider when we get copy from @PeloWriter.